packages/web-comment/src/routes/decided-appeals/index.njk

```diff
@@ -1,33 +1,33 @@
 {% extends "../../index.njk" %}
-{% set title = "Enter appeal reference" %}
```

Comment:
I think based on this prototype: https://appeals-prototype.herokuapp.com/interested/find-appeal/v9/closed-appeals?status=completed&appeal=1&appealCode=010234

The title is just meant to be "Decided appeals - ..."